stripe/src/main/java/com/stripe/android/view/MaskedCardView.java

```diff
@@ -225,7 +229,7 @@ private void updateCardInformation() {
         int last4length = mLast4.length();
         SpannableString str = new SpannableString(brandText + normalText + mLast4);
         str.setSpan(
-                new StyleSpan(Typeface.BOLD),
+                new TypefaceSpan("sans-serif-medium"),
```

Comment:
I was thinking I'd let design comment on it and change it then, since they'll have to once-over me on everything else (I'll point out the width difference)